### PR TITLE
Fix unbind API heading.

### PR DESCRIPTION
**Pull request by rajnesh-kanwal**

### CODE REVIEW

**Comment by rajnesh-kanwal** [comment]
cc: @sameo @rsahita

File: specification/sbi_atee.adoc [hunk 1]
```diff
@@ -1270,15 +1270,15 @@ The possible error codes returned in `sbiret.error` are shown below.
 == TEE Guest Extension (EID 0x54454547)
 The TEE Guest extension supplements the TEE Host extension, and allows TVMs to communicate with TSM.
 A typical use case for this extension is to relay information to the host. TEE-Guest calls cause a
-trap to the TSM which may exit to the host with `scause` set to ECALL, a6 set to FID, `a0`-`a5` set
-to ECALL args.
+trap to the TSM. TSM should do any processing required and then forward the ECALL to the host with
```
Comment (sameo) [inline specification/sbi_atee.adoc:1273]:
```suggestion
trap to the TSM. TSM should do any processing required and then must forward the ECALL to the host with
```

Comment (rajnesh-kanwal) [reply specification/sbi_atee.adoc]: Thanks. Fixed.

**Review by sameo — APPROVED** [review]

File: specification/sbi_atee.adoc [hunk 2]
```diff
@@ -1270,15 +1270,15 @@ The possible error codes returned in `sbiret.error` are shown below.
 == TEE Guest Extension (EID 0x54454547)
 The TEE Guest extension supplements the TEE Host extension, and allows TVMs to communicate with TSM.
 A typical use case for this extension is to relay information to the host. TEE-Guest calls cause a
-trap to the TSM which may exit to the host with `scause` set to ECALL, a6 set to FID, `a0`-`a5` set
-to ECALL args.
+trap to the TSM. TSM should do any processing required and then must forward the ECALL to the host with
```
Comment (atishp04) [inline specification/sbi_atee.adoc:1273]: Since we are clarifying this, let's add the a7 as the extension id part as well.

**Review by atishp04 — APPROVED** [review]
Apart from the nit comment:

File: specification/sbi_atee.adoc (hunk 2 above)
Comment (rajnesh-kanwal) [reply specification/sbi_atee.adoc:1273]: Fixed. Thanks.

**Review by rsahita — APPROVED** [review]

**Pull request merged**